### PR TITLE
Begin decompiling nusys

### PR DESCRIPTION
**Pull request by JoshDuMan**

### CODE REVIEW

**Comment by ethteck** [comment]
🐱

**Comment by deopdeop** [comment]
hi - LinuxTerm#0001

File: src/os/nusys/nuContInit.c [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#include "common.h"
+#include "nu/nusys.h"
+
+u8 nuContInit(void) {
+    u8 contBits;
```
Comment (ethteck) [inline src/os/nusys/nuContInit.c:5]: small thing but I prefer the initialization to happen on the same line it's declared if possible (as is in this case)

File: src/os/nusys/nuContRmbMgr.c [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+static s32 contRmbStopMesg(NUSiCommonMesg* mesg) {
+    NUContRmbMesg* rmbMesg = (NUContRmbMesg*) mesg->dataPtr;
+
+    nuContRmbCtl[rmbMesg->contNo].frame = 0;
+
+    return 0;
+}
+
+static s32 contRmbForceStopMesg(NUSiCommonMesg* mesg) {
+    u32 i;
```
Comment (ethteck) [inline src/os/nusys/nuContRmbMgr.c:144]: Other small thing, but I think we should use consistent types for loop itreators (assuming both u32 and s32 work) and we've been using s32 so far

File: src/os/nusys/nuGfxPreNMIFuncSet.c [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#include "common.h"
+#include "nu/nusys.h"
+
+void nuGfxPreNMIFuncSet(NUGfxPreNMIFunc func) {
+    OSIntMask intMask;
```
Comment (ethteck) [inline src/os/nusys/nuGfxPreNMIFuncSet.c:5]: Same line declaration thing here too

**Review by ethteck — CHANGES_REQUESTED** [review]

**Comment by nanaian** [comment]
Can probably fix those nits during merge / post-merge / without @JoshDuMan given its been ~20 days since review. @ethteck?

**Review by ethteck — APPROVED** [review]

**Review by pixel-stuck — APPROVED** [review]

**Pull request merged**